crates/networking/discv5/Cargo.toml

```diff
@@ -10,9 +10,18 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
 anyhow.workspace = true
 discv5.workspace = true
 futures.workspace = true
 libp2p.workspace = true
 tokio.workspace = true
 tracing.workspace = true
+
+# external crates 
+ethereum_ssz = "0.9.0"
+ethereum_ssz_derive = "1.0.0-beta.2"
+
+# ream dependencies
+ream-consensus.workspace = true
+ream-network-spec = { path = "../../common/network_spec" }
```

Comment:
![image](https://github.com/user-attachments/assets/36acd4c4-56b6-425d-88e2-028939518752)

this file is missing a new line at the end